### PR TITLE
Hotfix/23.4.3 SC-5048 disable portfolio

### PR DESCRIPTION
**Pull request by tofixx**
# Description

Temporarily disables Portfolio for NBC by removal of link to portfolio in add-ons.

## Links to Tickets or other pull requests
https://ticketsystem.schul-cloud.org/browse/SC-5048

## Deployment
added `FEATURE_EXTENSION_PORTFOLIO_ENABLED ` which defaults to `false`

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: theme/n21/views/addons/addons.hbs [hunk 1]
```diff
@@ -3,11 +3,13 @@
         <link rel="stylesheet" href="/styles/addons/addons.css"/>
     {{/content}}
     {{#content "page"}}
+      {{#ifConfig "FEATURE_EXTENSION_PORTFOLIO_ENABLED" true}}
```
Comment (ABialoskorski) [inline theme/n21/views/addons/addons.hbs:6]: could it be
`{{#if FEATURE_EXTENSION_PORTFOLIO_ENABLED }}` ?

Comment (tofixx) [reply theme/n21/views/addons/addons.hbs:6]: This needs to resolve the configuration, not an if: https://github.com/schul-cloud/schulcloud-client/blob/dcc73af86919fb7f0d665648be0546e90dd29aa9/helpers/handlebars/helpers/index.js#L139

Comment (ABialoskorski) [reply theme/n21/views/addons/addons.hbs:6]: ok :)

Comment (tofixx) [reply theme/n21/views/addons/addons.hbs:6]: No, this needs to resolve https://github.com/schul-cloud/schulcloud-client/blob/dcc73af86919fb7f0d665648be0546e90dd29aa9/helpers/handlebars/helpers/index.js#L139

**Comment by tofixx** [comment]
> If I get it right, the Add-Ons Sidebar Item will still be displayed, ending up in a page with no content, as the Portofolio tile is hidden?
> 
> I'd strongly suggest to hide the Sidebar AddOns Item in this case as well

As this page shows another entry currently I will keep the navigation item add-ons as it is
image

**Pull request merged**